### PR TITLE
Adding Jira team sprint quality metric

### PR DESCRIPTION
**Pull request by rudolf-repcin**

### CODE REVIEW

File: controllers/teamdetail.controller.js [hunk 1]
```diff
@@ -166,9 +162,24 @@ function getDateIntervals(startDate, endDate) {
...
+    if (error.response) {
+        return {
+            status: error.response.status,
+            message: error.response.data
+        }
+    }
+    else {
+        return {
+            status: 500,
+            message: `Couln't retrieve ${metricType}. ${error.message}`
```
Comment (olgazavodska) [inline controllers/teamdetail.controller.js]: small typo

File: middleware-spec.yaml [hunk 2]
```diff
@@ -49,7 +54,9 @@ paths:
   /team-detail/jira:
     get:
       summary: JIRA data for specific team
-      description: JIRA data for specific team and given period of time. Default time period is last 6 months.
+      description: >-
+        JIRA data for specific team and given period of time. Default time
```
Comment (olgazavodska) [inline middleware-spec.yaml]: Could you please add list of available metrics to the description?

File: middleware-spec.yaml [hunk 3]
```diff
@@ -75,23 +84,25 @@ paths:
...
             application/json:
               schema:
-                $ref: '#/components/schemas/error'
+                $ref: '#/components/schemas/authError'
   /team-detail/github:
     get:
       summary: Github data for specific team
-      description: Github data for specific team and given period of time. Default time period is last 6 months.
+      description: >-
+        Github data for specific team and given period of time. Default time
```
Comment (olgazavodska) [inline middleware-spec.yaml]: Could you please add list of available metrics to the description?

File: controllers/teamdetail.controller.js (hunk 1 above)
Comment (rudolf-repcin) [reply controllers/teamdetail.controller.js]: Yes I noticed :)

**Pull request merged**